rpa-adapter/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+  id 'net.ltgt.apt' version '0.14'
+  id 'jacoco'
+}
+
+def springBootVersion = '2.0.1.RELEASE'
+
+def versions = [
+  springBoot: springBootVersion
+]
+
+dependencies {
+  compile project(':domain-model')
+
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter', version: versions.springBoot
+  compile group: 'javax.json', name: 'javax.json-api', version: '1.1'
+  compile group: 'org.glassfish', name: 'javax.json', version: '1.1'
+
+  testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: versions.springBoot
+  testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
does junit not come in the spring-boot starter?

Reply:
Was about to do that cleanup from initial @dharmendrak version.